docs/catalog/cloud/remove-unused-assets.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Remove unused assets
+
+## Description
+
+Monitor and analyze the application and the cloud bill to identify resources that are no longer used or can be reduced.
+
+## Solution
+
+Decommission unused assets to free consumed resources. Consolidate resources, for example different database instances with a only a few tables could be moved into one instance. Review generated assets if they contain redundant content and handle them accordingly. 
+Inspect storage buckets if unused assets can be removed.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+With respect to the SCI equation, remove unused assets will impact one part:
+
+- `E`: Reduces the embodied emissions by not using the resources
```

Comment:
Should we include `M` here to account for the saved embodied carbon in storage resources after removing the assets?

Reply:
Just noticing that E is wrong :(